src/blog/2023/10/blueprints.md

```diff
@@ -0,0 +1,37 @@
+---
+title: What are FlowFuse Blueprints?
+subtitle: Preconfigured Node-RED Applications
+description: Explore the concept of FlowFuse Blueprints, the pre-built solutions to simplify Node-RED experience, and take a look at the first three Blueprints we've launched for manufacturing applications.
+date: 2023-10-16
+authors: ["marian-demme"]
+#image: "/blog/2023/10/images/ANDON1.png"
+tags:
+    - posts
+    - releases
+    - flowfuse
+    - changelog
+---
+
+Starting today, FlowFuse Blueprints are available on FlowFuse Cloud. Additionally, upon request, all our Teams and Enterprise Self-Hosted customers gain access to this collection. But what exactly are FlowFuse Blueprints?
+
+<!--more-->
+
+## FlowFuse Blueprints
+
+FlowFuse Blueprints aim to make the Node-RED experience more accessible for newcomers, while also offering a treasure trove of fresh ideas for seasoned Node-RED users. When setting up a new Node-RED instance, you now have the option to choose a Blueprint tailored for specific use cases. For example, our "ANDON Operator Terminal" Blueprint can be selected, and it will automatically configure the Node-RED instance, sparing you the need to start from scratch. While these templates are powerful out-of-the-box, they're also fully customizable, allowing you to tweak them to suit your unique requirements. Ultimately, Blueprints speed up the learning curve for new users and expedite the solution-building process for experienced ones.
```

Comment:
```suggestion
FlowFuse Blueprints aim to make the Node-RED experience more accessible for newcomers, while also offering a treasure trove of fresh ideas for seasoned Node-RED users. When setting up a new Node-RED instance, you now have the option to choose a blueprint tailored for specific use cases. For example, our "ANDON Operator Terminal" blueprint can be selected, and it will automatically configure the Node-RED instance, sparing you the need to start from scratch. While these templates are powerful out-of-the-box, they're also fully customizable, allowing you to tweak them to suit your unique requirements. Ultimately, blueprints speed up the learning curve for new users and expedite the solution-building process for experienced ones.
```